postreise/analyze/generation/capacity_value.py

```diff
@@ -97,3 +97,17 @@ def get_storage_capacity(scenario, area, area_type=None):
     storage_id = get_storage_id_in_area(scenario, area, area_type)
 
     return grid.storage["gen"].loc[storage_id].Pmax.sum()
+
+
+def sum_capacity_by_type_zone(scenario):
+    """Sum generator capacity for a Scenario in Analyze state by {type, zone}.
```

Comment:
Does the scenario need to be in the analyze state?Like the above two functions we just need to have a `Grid` object that can be obtained in any state.

Reply:
Good catch. No. It can be at any state. All it needs the generator capacity from the grid.

Reply:
Done.